spec/hyacinth/adapters/external_identifier_adapter/datacite/rest_api/v2/data_spec.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Hyacinth::Adapters::ExternalIdentifierAdapter::Datacite::RestApi::V2::Data do
+  let(:data) { described_class.new('10.33555') }
```

Comment:
Maybe worth adding a test that verifies initializer arguments create an object with the expected defaults. (e.g. passing in doi prefix actually sets prefix internally)

Reply:
Test added.